tools/test_history.py

```diff
@@ -223,6 +229,10 @@ def display_history(
     )
     args = parser.parse_args()
 
+    jobs = None if args.all else args.job
+    if jobs == []:  # no jobs, and not None (which would mean all jobs)
+        sys.exit('No jobs specified.')
```

Comment:
raise argparse.ArgumentError("...")

Reply:
@walterddr what are the benefits of that over using `sys.exit` here?

Reply:
oh, or did you mean `parser.error`? I can see that being beneficial, since it causes the script usage info to be printed